R/extend_scenario_trajectory.R

```diff
@@ -73,8 +73,7 @@ extend_scenario_trajectory <- function(data,
     dplyr::inner_join(
       scenario_data,
       by = c("ald_sector", "technology", "scenario_geography", "scenario", "year")
-    ) %>%
-    dplyr::distinct_all()
+    )
```

Comment:
just to go sure, there is no possible scenario where the distinct_all here is needed? When in doubt I would recommen using report_all_duplicate kinds on it. The fucntion has been changed to invisibly return the input, so you could simply pipe it in with %>% reprt_all_duplicate_kinds(names(.))

Reply:
done

Reply:
to me it looked like you havent. Or just not in this fucniton because uneccesray?

Reply:
imo direction is not part of CUC

Reply:
you are right!